meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx6ullevk/lmp.cfg

```diff
@@ -74,15 +84,9 @@ CONFIG_ENV_IS_NOWHERE=y
 # CONFIG_ENV_IS_IN_MMC is not set
 # CONFIG_ANDROID_BOOT_IMAGE is not set
 # CONFIG_NET is not set
-# CONFIG_CMD_NET is not set
-# CONFIG_CMD_DHCP is not set
-# CONFIG_CMD_BOOTP is not set
-# CONFIG_CMD_TFTPBOOT is not set
-# CONFIG_CMD_NFS is not set
```

Comment:
Why removing that?

Reply:
These features are disabled due to `# CONFIG_NET is not set` above.  These are redundant at that point.